src/Containers/Tag.tsx

```diff
@@ -6,18 +6,20 @@ import { Main } from '../Utils/BaseStyles';
 
 export interface TagProps extends React.HTMLAttributes<HTMLSpanElement> {
     icon?: React.ForwardRefExoticComponent<React.RefAttributes<SVGSVGElement>>;
+    hasHoverMode?: boolean;
     children: React.ReactNode;
 }
 
 export const Tag: React.FC<TagProps> = ({
     icon = Times,
     children,
+    hasHoverMode = true,
     ...props
 }): React.ReactElement => {
     return (
         <TagDiv {...props}>
             {children}
-            <Icon as={icon} />
+            {!!hasHoverMode && <Icon as={icon} />}
```

Comment:
This code is a bit confusing, the icon only shows if what has hoverMode, the tag? What does the icon has to do with the styling has Hover mode?
Also, isHoverable is a better name ;)